src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -2248,6 +2248,15 @@ public void ImportLibrary(object parameter)
                     foreach (var file in openFileDialog.FileNames)
                     {
                         EngineController.ImportLibrary(file);
+                        string path = new FileInfo(file).Directory.FullName;
```

Comment:
does `ImportLibrary` get called from anywhere else expect the UI method?

Reply:
It appears that `ImportLibrary` is only called here and in one test file.

Reply:
Correction; `EngineController.ImportLibrary` is only called here and in one test, but looking a little deeper `EngineController.ImportLibrary` just calls `LibraryServices.ImportLibrary`, which is called from `ZeroTouchNodeLoader.CreateNodeFromXml` and `DynamoModel.LoadNodeLibrary`. `DynamoModel.LoadNodeLibrary` is called from the package manager so I think it may be decoupled from the functionality being implemented here, due to the paths already being present in order for the package manager to be able to load the library.